base/libgit2/libgit2.jl

```diff
@@ -489,10 +489,15 @@ function rebase!(repo::GitRepo, upstream::AbstractString="")
         else
             GitAnnotated(repo, upstream)
         end
+        onto_ann  = if isempty(newbase)
+                        Nullable{GitAnnotated}()
+                    else
+                        Nullable{GitAnnotated}(GitAnnotated(repo, newbase))
+        end
```

Comment:
This seems stylistically different than what's around elsewhere in Base (or maybe it's just that the `else` and `end` aren't in the same column that's bugging me). Perhaps something like
```julia
onto_ann = if isempty(newbase)
    Nullable{GitAnnotated}()
else
    Nullable{GitAnnotated}(GitAnnotated(repo, newbase))
end
```
would be acceptable? Not really a big deal.